plugin/flags_sources/compilation_db.py

```diff
@@ -30,14 +31,17 @@ class CompilationDb(FlagsSource):
     """
     _FILE_NAME = "compile_commands.json"
 
-    def __init__(self, include_prefixes):
+    def __init__(self, include_prefixes,
+                 header_to_source_map=["{stamp}.*"]):
```

Comment:
Same here, we could probably avoid the default variables as they tend to strike in the back when one least expects it :wink: